<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LePoids</class>
 <widget class="QMainWindow" name="LePoids">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>545</width>
    <height>466</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LePoids</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>421</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Informations personnelles</string>
     </attribute>
     <widget class="QSpinBox" name="spinBoxAge">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>42</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelAge">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxPoids">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>62</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <double>15.000000000000000</double>
      </property>
      <property name="maximum">
       <double>300.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="labelTaille">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Taille (m)</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSexe">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sexe</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNom">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPrenom">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPoids">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Poids (kg)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonHomme">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Homme</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonFemme">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Femme</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNom">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>30</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPrenom">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSuite">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>230</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Suite</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxTaille">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>62</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <double>0.500000000000000</double>
      </property>
      <property name="maximum">
       <double>3.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Calcul du poids idéal</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonDevine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>301</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Poids idéal formule de devine</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLorentz">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>301</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Poids idéal formule de lorentz</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAge">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>301</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Poids idéal formule de lorentz en fonction de l'age</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditAfficheur">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>311</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonQuitter">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>LePoids</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
